openbas-api/src/main/java/io/openbas/rest/mapper/form/MapperAddInput.java

```diff
@@ -0,0 +1,31 @@
+package io.openbas.rest.mapper.form;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Pattern;
+import lombok.Getter;
+import lombok.Setter;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static io.openbas.config.AppConfig.MANDATORY_MESSAGE;
+
+@Getter
+@Setter
+public class MapperAddInput {
```

Comment:
I think we need to remove it right @Dimfacion ?

Reply:
Yes, I'm on it